### ISSUE TITLE
Can this integrate with tensor flow?

### ISSUE THREAD

**Issue by NullVoxPopuli** [studioLaCosaNostra/ml-double-q-learning#1]
Hello, I'm still learning how reinforcement algorithms tie in to neural networks, and am wondering if/how this could tie in to a tensor flow neural network / layers model?

**Comment by rayros** [comment]
I don't think so. Neural matrixes are formed in Tensorflow. There's no such thing here.

**Comment by NullVoxPopuli** [comment]
I mean, the q lookup table is essentially a function yeah? Same with a neural network. The inputs just need massaging?
